### DISCUSSION TITLE
记录 所有的req_headers

### DISCUSSION THREAD

**Discussion by arrow2012** [bfenetworks/bfe#1169] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

在日志中要记录 请求头 ，使用 ${User_Agent}req_header 这种指定的变量 来记录指定的req_Header
但是在生产环境中 可能 用户有很多请求头，并且不是固定的，
nginx 可以通过 lua脚本来 获取req_headers  中的所有header

BFE 能否记录所有的req_Header Key和value 如下图 
![image](https://user-images.githubusercontent.com/46478986/125940596-04bdfa9d-437f-4afe-9002-c57072176ae3.png)

**Comment by iyangsj** [comment]
@arrow2012 感谢你的关注和反馈！后续版本会考虑支持
